internal/isoeditor/rhcos.go

```diff
@@ -43,12 +43,23 @@ func RamdiskImageArchive(netFiles []staticnetworkconfig.StaticNetworkConfigData,
 				return nil, err
 			}
 		}
-		scriptPath := "/usr/lib/dracut/hooks/initqueue/settled/90-assisted-pre-static-network-config.sh"
-		scriptContent := constants.PreNetworkConfigScript
 
+		scriptPath := "/usr/sbin/assisted-static-network-config.sh"
+		scriptContent := constants.PreNetworkConfigScript
 		if err := addFileToArchive(w, scriptPath, scriptContent, 0o755); err != nil {
 			return nil, err
 		}
+
+		servicePath := "/usr/lib/systemd/system/assisted-static-network-config.service"
+		serviceContent := constants.PreNetworkConfigService
+		if err := addFileToArchive(w, servicePath, serviceContent, 0o644); err != nil {
+			return nil, err
+		}
+
+		serviceLink := "/etc/systemd/system/initrd.target.wants/assisted-static-network-config.service"
```

Comment:
Is this only needed in the minimal ISO case?

We don't provide anything that runs during initrd in the full ISO so I'd be surprised if this worked there.

Reply:
I didn't check the full ISO, will do.

Reply:
The full ISO doesn't need this: the network isn't started early because there is no rootfs URL in the kernel command line arguments.